### DISCUSSION TITLE
Own-/Nextcloud synchronisiert nicht mehr: unsupported server version

### DISCUSSION THREAD

**Discussion by CorporalCane** [tine20/tine20#7460] [Q&A] [answered]
Hallo miteinander,

seit geraumer Zeit finde ich keine Lösung für das o.g. Problem.
neuere Versionen von Own-/Nextcloud (ich meine ab Version 3.2) versuchen über den Link  https://server/dav  -statt: https://server/webdav-  auf Tine zuzugreifen. Webdav im Browser funktioniert. DAV geht nicht.
Owncloud Version 2.5.1 funktioniert sowohl unter Linux, als auch unter Windows.
Beim Link  https://server/dav  kommt keine Fehlermeldung (im Browser), sondern der Bildschirm bleibt leer.

Mein Server läuft unter Debian Bullseye, Kernel 5.10.0-22, PHP 8.2.6.

Hat jemand eine Idee, was ich ändern kann/muss, damit die Synchronisation auch wieder mit aktuellen Versionen läuft?

Ich bin für jeden Tip dankbar!

Viele Grüße
Markus

**Comment by lab-at-nohl** [comment]
Hast du [tineDrive](https://github.com/tine20/tineDrive/releases/) probiert? Das ist ein Fork von Owncloud-client. Letzte Version ist halt auch nur 3.2.1 oder so, aber wurde vor zwei Wochen erst aktualisiert. Was fehlt dir? Oder suchst du nach einer Handy App?

Wenn es nur um das Verzeichnis `/dav` ginge, könntest du einfach eine rewriteRule hinzufügen (alte owncloud clients rufen m.E. `/remote.php/webdav` o.ä. auf). Wie hängt davon ab, ob du Apache2 oder nginx nutzt. Ich glaube aber, das Problem ist tiefergehend.

VG

**Reply by lab-at-nohl** [reply]
Ok, tried myself. Latest tineDrive 3.2.1 is looking for `"GET /ocs/v2.php/cloud/user?format=json HTTP/1.1" 404`, which is not available (404) with Tine of course. Obviously, Next Cloud invented another protocol beside webdav for management. 

Saying this, you need another WebDAV client, other than ownCloud or Next Cloud.

**Comment by lab-at-nohl** [comment, marked as answer]
Hallo CorporalCane, 

um einen Stricht drunter zu machen. Mit einem jüngeren [Commit](https://github.com/tine20/tine20/commit/0fdb143d5a66b2a4ef589eac0e4406a7ad377814), der allerdings noch in keinem Release auftaucht, wurde OwnCloud 3-Support eingeführt. Dann funktioniert tineDrive (theoretisch, unter Linux stürzt es bei mir ab), aber auch der Client von Nextcloud (aktuell 3.8.x - mit Hinweis, dass Servertyp unbekannt).

Wenn du weißt, was du tust, kannste den Patch bei dir umsetzen. Kein besonders tiefer Eingriff; Hinweis: Danach Cache löschen, sonst wird die neue URL nicht bedient...

VG

**Reply by pschuele** [reply]
ein release mit dem fix ist nun auch verfügbar: https://github.com/tine20/tine20/releases/tag/2023.05.30.960

**Reply by CorporalCane** [reply]
Hallo lab-at-nohl,

tineDrive habe ich nicht probiert, aber das Commit eingepflegt.
OpenSuse mit OwnCloud 3.8.0 klappt wieder
Win10 mit owncloud 3.6.2 klappt
Win10 mit owncloud 4.0 (Upgrade von 3.6.2) läuft bei mir nicht. ich schaue noch nach. Ggf. habe ich da noch was falsch gemacht
Win10 mit Nextcloud 3.8.2 (Neuinstallation) läuft
Win11 mit Nextcloud 3.6.2 + 3.8.2 (Update) läuft ebenfalls
Die Meldung mit nicht unterstützter Server-Version ignoriere ich erstmal ;)

Das Release 2023.05.30.960 werde ich vermutlich erst im Juli/Aug. testen können. Aktuell bin ich froh die o.g. Rechner synchronisieren zu können.

Vielen Dank!
